src/ApiGenerator/Generator/Razor/DescriptorsGenerator.cs

```diff
@@ -12,6 +13,10 @@ public class DescriptorsGenerator : RazorGeneratorBase
 
 		public override async Task Generate(RestApiSpec spec, ProgressBar progressBar)
 		{
+			// Delete existing files
+			foreach (var file in Directory.GetFiles(GeneratorLocations.NestFolder, "Descriptors.*.cs"))
```

Comment:
pull out `"Descriptors"` into a `const` e.g.

```c#
const string Descriptors = nameof(Descriptors);
```

Might be good to for all generators